tests/microos/journal_check.pm

```diff
@@ -37,7 +37,7 @@ sub run {
         bsc_1118321         => '.*update-checker-migration.timer.*(Failed to parse calendar specification|Timer unit lacks value setting).*',
         bsc_1126272         => 'Failed unmounting \/\S+\.|-- Reboot --|pam_systemd.*Failed to release session',
         bsc_1127339         => 'kernel: efi: EFI_MEMMAP is not enabled',
-        bsc_000000_FEATURE  => 'health-checker/rebootmgr.sh check" failed|Machine didn\'t come up correct, do a rollback',
+        bsc_000000_FEATURE  => '/usr/lib/health-checker/fail.sh check',
```

Comment:
On Tumbleweed, it's `libexec` meanwhile. So this should include the same `health-checker/fail.sh check" failed` as before.

The second match `Machine didn\'t come up correct, do a rollback` is also still needed

Reply:
right, I didn't notice, just looked at the journal in TB:
```
Oct 06 19:06:53.900310 localhost root[593]: ERROR: "/usr/libexec/health-checker/fail.sh check" failed
Oct 06 19:06:53.900695 localhost health-checker[593]: <10>Oct  6 19:06:53 root: ERROR: "/usr/libexec/health-checker/fail.sh check" failed
Oct 06 19:06:53.992594 localhost health-checker[548]: Health check failed!
```